os/net/lwip/configs/Kconfig

```diff
@@ -227,24 +227,6 @@ config NET_LWIP_SNMP_MAX_TREE_DEPTH
 
 endif #NET_LWIP_SNMP
 
-menu "Interface Name"
-config NET_ETH_IFNAME
```

Comment:
If these configs are not needed anymore, how about removing them from defconfigs also?

Reply:
I'll modify it next PR